liquibase-service/src/main/resources/db/rdb_modern/routines/017-sp_d_labtest_result_postprocessing-002.sql

```diff
@@ -1184,11 +1184,51 @@ BEGIN
 
         COMMIT TRANSACTION;
 
+
+        BEGIN TRANSACTION;
+
+        SET @PROC_STEP_NO =  @PROC_STEP_NO + 1 ;
+        SET @PROC_STEP_NAME = 'DELETE LAB_RESULT_COMMENT ';
+
```

Comment:
If the key exists, the uid associated is removed using the #tmp_result_comment_group. 

That being said, the uid is also excluded from #TMP_New_Lab_Result_Comment_FINAL because of the lack of association from nrt_observation_txt?

Reply:
Yes, the logic of exclusion of the uid from the #TMP_New_Lab_Result_Comment_FINAL is not changed, that table is empty if the comment was deleted.